doc/docusaurus/docs/using-plinth/evaluating-plinth.md

```diff
@@ -0,0 +1,201 @@
+---
+sidebar_position: 12
+---
+
+# Evaluating CompiledCode for testing
+
+The `CompiledCode` is intended to be evaluated by Cardano nodes when 
+transaction validation occurs. For this purpose, it is serialized and included in a transaction.
+
+However, it is also possible to evaluate `CompiledCode` in a test environment. 
+This is useful for testing and troubleshooting. By doing so, Plinth developers can
+not only get the immediate result of the code but also obtain the traces emitted 
+during the evaluation, as well as the consumed execution budget.
+
+Let's consider the following example Plinth program:
+<LiteralInclude 
+  file="Example/Evaluation/Main.hs" 
+  language="haskell" 
+  title="Example Plinth code" 
+  start="-- BEGIN Plinth" 
+  end="-- END Plinth" />
+
+In order to evaluate this code, we need to add the `plutus-tx:plutus-tx-testlib` 
+dependency to our cabal file:
+```cabal
+  build-depends:
+    , plutus-tx:plutus-tx-testlib
+```
+
+So that we can import the necessary functionality:
+
+```haskell
+import PlutusTx.Test (
+  CodeResult,
+  applyLifted,
+  evaluateCompiledCode,
+  prettyCodeResult
+ )
+```
+
+Our example code represents a function that expects an `Integer` argument
+and returns the `Integer` value. 
+
+It is possible to evaluate this compiled code without applying it to any arguments, 
+and evaluation will succeed, returning the value of type `Integer -> Integer`:
+
+```haskell
+result :: CodeResult
+result = evaluateCompiledCode compiledCode
+```
+
+The `CodeResult` type is declared as follows:
+
+```haskell
+data CodeResult = CodeResult
```

Comment:
Can you `LiteralInclude` this?

Reply:
It would have to reference the file `plutus-tx/testlib/PlutusTx/Test/Run/Code.hs` outside of the `doc` folder; This would make the `doc/docusaurus` not self-contained. 

(I gave it a try anyway, and it didn't work: `Error loading code block`)